pkg/koordlet/resourceexecutor/updater.go

```diff
@@ -486,3 +489,93 @@ func commonWriteIfDifferentWithLog(c *DefaultResourceUpdater) error {
 	}
 	return nil
 }
+
+func BlkIOUpdateFunc(resource ResourceUpdater) error {
+	info := resource.(*CgroupResourceUpdater)
+	audit.V(5).Group("blkio").Reason("UpdateBlkIO").Message("update %v to %v", info.file, info.value).Do()
+	return cgroupBlkIOFileWriteIfDifferent(info.parentDir, info.file, info.Value())
+}
+
+// resourceType sysutil.ResourceType, parentDir string, value string, updateFunc UpdateFunc
+func NewBlkIOResourceUpdater(file sysutil.Resource, value string, parentDir string) *CgroupResourceUpdater {
```

Comment:
Could we register the cgroup updater of the blkio resource in DefaultCgroupUpdaterFactory to support cgroups-v2 and maintain the blkio updater uniformly? Afterward, the callers can use the blkio updater via `DefaultCgroupUpdaterFactory.New(resourceType, cgroupParent, value, auditor)`.

Reply:
done